langwatch/src/components/filters/FieldsFilters.tsx

```diff
@@ -60,9 +64,32 @@ export function FieldsFilters() {
     availableFilters[id],
   ]);
 
+  const nonEmptyFilters = Object.values(filterParams.filters).filter((f) =>
+    typeof f === "string"
+      ? !!f
+      : Array.isArray(f)
+      ? f.length > 0
+      : Object.keys(f).length > 0
+  );
+
+  const hasAnyFilters = nonEmptyFilters.length > 0;
```

Comment:
hmm this logic is duplicated from `FilterToggle` right? What if we move it inside `useFilterParams`? expose it as a new key on the return object `hasAnyFilters`, which you can use from here and FilterToggle